### PR TITLE
[[Documentation]] keyword/long - detailed|long

### PR DESCRIPTION
**Pull request by seaniepie**
- camelCase from defaultfolder to defaultFolder
- Describe relationship between 'long' and 'detailed' with script formatting.

### CODE REVIEW

**Comment by seaniepie** [comment]
There seems to be a mismatch in references to the 'long' keyword in the files and folders functions documentation. There is also no link from them to 'file(keyword)' and 'folder(keyword)' and vice versa. Does this need to be addressed?

**Comment by livecodeali** [comment]
@seaniepie If by a mismatch you mean that the `long` keyword should be referenced in the `files` and `folders` function entries, then yes I agree.

The `file` and `folder` keywords probably shouldn't have links to the `long` keyword, as they are referring to file and folder in the sense of `there is a file | folder` which doesn't relate to the `long` keyword.

**Comment by seaniepie** [comment]
1. There is not a 'Folder' keyword dictionary entry. Does this mean that the word folder is not a keyword even when used in the `there is a folder` form.
2. 'File' keyword is only referred to in the lcdoc for it in reference to `URL "file:*.*"`, again not for `there is a file`.
3. What I meant, to clarify, is that in both the 'function/files' and 'function/folders' docs there is no reference 'linking' to this 'keyword/long' doc or 'detailed'. You've answered that tho so I'll fix both of those too.

**Comment by livecodeali** [comment]
@seaniepie Ah yes, right you are!

**Comment by seaniepie** [comment]
@seaniepie Link files and folders to this and vice versa

**Comment by seaniepie** [comment]
@livecodeali , the time doc has references to `<format|formatted>` but it has links to both the function and glossary in the references section. Does the tag need defining or does it go to the first link by default?

**Comment by livecodeali** [comment]
@seaniepie The `format` function is the one that creates strings with substituted parameters and so on, which I think is completely irrelevant to the `time` function, so that related item should be removed entirely. All uses of the word `format` are just to do with the concept of format rather than the function as far as I can tell!

**Comment by seaniepie** [comment]
👍

File: docs/dictionary/keyword/long.lcdoc [hunk 1]
```diff
@@ -26,36 +26,39 @@ Example:
...
 
-A long object name consists of the <object type>, followed by the name
+A long object <name> consists of the <object type>, followed by the name
 in quotes, for the <object(glossary)> and its <owner> (and the <owner>
-of that <object(glossary)>, and so on). For example, a long <card> name
+of that <object(glossary)>, and so on). For example, a long <card> <name>
 looks like this: card "My Card" of stack "/Drive/Folder/Stack.rev"
 
-A long object ID looks like this:
+A long object <ID> looks like this:
```
Comment (livecodeali) [inline docs/dictionary/keyword/long.lcdoc]: These are case sensitive IIRC.

Comment (livecodeali) [inline docs/dictionary/keyword/long.lcdoc]: This should be directed at the instance in the references list, sorry 😄

File: docs/dictionary/keyword/long.lcdoc [hunk 2]
```diff
@@ -69,7 +72,8 @@ propertyNames (function), time (function), folders (function),
 files (function), dateFormat (function), date (function),
 keyword (glossary), property (glossary), command (glossary),
 object type (glossary), expression (glossary), object (glossary),
-card (keyword), short (keyword), abbreviated (keyword), owner (property)
+card (keyword), short (keyword), abbreviated (keyword), id (property)
```
Comment (livecodeali) [inline docs/dictionary/keyword/long.lcdoc]: This needs to be `ID` rather than `id`.

Comment (seaniepie) [reply docs/dictionary/keyword/long.lcdoc]: Got it.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 897cec7

**Comment by livecodeali** [comment]
Thanks @seaniepie!

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 897cec717bb862af3f1eb2a0b4f6fb971be5d980

**Comment by livecode-vulcan** [comment]
😞 test failure 897cec717bb862af3f1eb2a0b4f6fb971be5d980
- try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/503)
- try-community-mac: [pending](https://vulcan.livecode.com/builders/try-community-mac/builds/508)
- try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/503)
- try-community-emscripten: [failure](https://vulcan.livecode.com/builders/try-community-emscripten/builds/503)
- try-community-ios: [pending](https://vulcan.livecode.com/builders/try-community-ios/builds/509)
- try-community-win-x86: [pending](https://vulcan.livecode.com/builders/try-community-win-x86/builds/503)
- try-community-android-armv6: [failure](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/503)

**Comment by seaniepie** [comment]
Has this emscripten/android fail been fixed now. I saw @montegoulding was working on it today.

**Comment by montegoulding** [comment]
I think @peter-b is working on it and it's still a thing

**Comment by seaniepie** [comment]
@montegoulding Travis should have picked up on something in the 'files' doc. In the syntax we have a parameter, `<targetFolder>`, which is not in a parameters section. I'm adding a param section to it now but it's interesting that CI wasn't able to pick it up.

**Comment by seaniepie** [comment]
Folders doc had the same issue.

**Comment by montegoulding** [comment]
@seaniepie the develop branch doesn't have the full range of tests that my validatedocs branch does. There are errors about this on the validatedocs branch I believe.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 24aa3d8

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 24aa3d84fa3e54a2b8898240a7b187b8e3a2ee15

**Comment by livecode-vulcan** [comment]
😞 test failure 24aa3d84fa3e54a2b8898240a7b187b8e3a2ee15
- try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/522)
- try-community-mac: [pending](https://vulcan.livecode.com/builders/try-community-mac/builds/527)
- try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/522)
- try-community-emscripten: [failure](https://vulcan.livecode.com/builders/try-community-emscripten/builds/526)
- try-community-ios: [pending](https://vulcan.livecode.com/builders/try-community-ios/builds/531)
- try-community-win-x86: [pending](https://vulcan.livecode.com/builders/try-community-win-x86/builds/523)
- try-community-android-armv6: [failure](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/523)

**Comment by montegoulding** [comment]
@livecode-vulcan try 24aa3d8

**Comment by livecode-vulcan** [comment]
😎 test success 24aa3d84fa3e54a2b8898240a7b187b8e3a2ee15
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/548)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/553)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/548)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/552)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/557)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/549)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/549)

**Pull request merged**